packages/react-native-bpk-component-calendar/src/common-types.js

```diff
@@ -106,4 +109,5 @@ export const commonDefaultProps = {
   selectionType: SELECTION_TYPES.single,
   style: null,
   disabledDates: null,
+  colorBuckets: undefined,
```

Comment:
Why `undefined` and not `null`?

Reply:
undefined is the same as not providing the prop whereas null will add the prop with a null value. it's manly to not pollute snapshots